Detail/ObjectView.swift

```diff
@@ -64,60 +64,65 @@ class ObjectView: UIView {
-            ])
+        for (index, component) in components.enumerated() {
+            let componentView = getViewComponent(from: component, in: self)
 
-            switch index {
-            case 0:
+            if let componentView = componentView {
                 NSLayoutConstraint.activate([
-                    componentView.topAnchor.constraint(equalTo: contentView.topAnchor, constant: .mediumLargeSpacing),
+                    componentView.leadingAnchor.constraint(equalTo: contentView.leadingAnchor, constant: .mediumLargeSpacing),
+                    componentView.trailingAnchor.constraint(equalTo: contentView.trailingAnchor, constant: -.mediumLargeSpacing),
                 ])
-            case components.count-1:
-                guard let previousComponentView = previousComponentView else {
-                    fatalError()
-                }
-                NSLayoutConstraint.activate([
-                    componentView.topAnchor.constraint(equalTo: previousComponentView.bottomAnchor, constant: .mediumLargeSpacing),
-                    componentView.bottomAnchor.constraint(equalTo: contentView.bottomAnchor, constant: -.mediumLargeSpacing),
-                ])
-            default:
-                guard let previousComponentView = previousComponentView else {
-                    fatalError()
+
+                switch index {
+                case 0:
+                    NSLayoutConstraint.activate([
+                        componentView.topAnchor.constraint(equalTo: contentView.topAnchor, constant: .mediumLargeSpacing),
+                    ])
+                case components.count-1:
+                    guard let previousComponentView = previousComponentView else {
+                        fatalError()
+                    }
+                    NSLayoutConstraint.activate([
+                        componentView.topAnchor.constraint(equalTo: previousComponentView.bottomAnchor, constant: .mediumLargeSpacing),
+                        componentView.bottomAnchor.constraint(equalTo: contentView.bottomAnchor, constant: -.mediumLargeSpacing),
+                    ])
+                default:
+                    guard let previousComponentView = previousComponentView else {
+                        fatalError()
+                    }
+                    NSLayoutConstraint.activate([
+                        componentView.topAnchor.constraint(equalTo: previousComponentView.bottomAnchor, constant: .mediumLargeSpacing),
+                    ])
                 }
-                NSLayoutConstraint.activate([
-                    componentView.topAnchor.constraint(equalTo: previousComponentView.bottomAnchor, constant: .mediumLargeSpacing),
-                ])
+                previousComponentView = componentView
+            }
+        }
+    }
+
+    func getViewComponent(from component: Component, in objectView: ObjectView) -> UIView? {
```

Comment:
Maybe better to call it:

`func viewComponent(for component: Component, in objectView: ObjectView) -> UIView? {`